packages/atlas/src/views/studio/VideoWorkspace/NftForm/NftForm.styles.ts

```diff
@@ -21,16 +21,17 @@ export const NftWorkspaceFormWrapper = styled.div`
 export const NftPreview = styled.div`
   background: black;
   display: flex;
+  gap: ${sizes(4)};
```

Comment:
We shouldn't use `gap` when the container display is set to flex.

Reply:
right